changelog/unreleased/issue-4934

```diff
@@ -0,0 +1,9 @@
+Enhancement: Clear removed snapshots, index and pack files from the local cache
```

Comment:
`index` and `pack` files already get removed from the cache before this PR. Thus the should not be mentioned in the changelog entry.